bin/crew

```diff
@@ -1308,6 +1308,9 @@ def resolve_dependencies
   # leave only dependency names (remove all package attributes returned by @pkg.get_deps_list)
   @dependencies.map!(&:keys).flatten!
 
+  # abort if we have incompatible dependencies
+  abort "Some dependencies are not compatible with your device architecture (#{ARCH}). Skipping install".lightred if @dependencies.any? { |dep| !Package.load_package("#{CREW_PACKAGES_PATH}/#{dep}.rb").compatible? }
```

Comment:
This is not isolated to just installs but builds also.  Maybe change `Skipping install` to `Unable to continue`?

Reply:
It might be helpful to list the dependencies which are incompatible.

Reply:
We don't know what they are-- It's possible to rework the code so that we do, but that'll slow things down.

Besides, this is only there in a worst-case scenario, we should only hit this code if someone forgets to change compatibility for a package.